charts/kuberpult/ci/test-values-override.yaml

```diff
@@ -0,0 +1,19 @@
+git:
```

Comment:
What is the use of this file?

Reply:
some values are required in the chart, these cannot be added in values.yaml.tpl since those need to be provided by the end user. Other values like memory and cpu are changed because the kind cluster in the pipeline is underpowered, so memory and cpu is reduced in pipeline.

So the overrides are applied on top of values.yaml to create test-values.yaml.